frontend/src/expert/components/SidePanelCard.js

```diff
@@ -184,13 +184,13 @@ function SidePanelCard(props) {
         // Code based on
         //   http://stackoverflow.com/questions/1064089/inserting-a-text-where-cursor-is-using-javascript-jquery
         const code = document.getElementById('code-editor');
-        if (document.selection) {
-            code.focus();
-            var sel = document.selection.createRange();
-            sel.text = text;
+        if (document.selection) { // if text in the code text-field is selected
             code.focus();
+            var sel = document.selection.createRange(); // creates a fragment of the text-field's value
+            sel.text = text; // replaces the fragment with the new text
+            code.focus(); // returns the cursor to the code text-field
```

Comment:
Can you explain what this change does?

Reply:
There was no change to the code here, just some annotations

Reply:
It looks like there's an additional code.focus()

Reply:
@LiteralMineral ?

Reply:
oh sorry I see now nevermind!